test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -5528,9 +5510,11 @@ BOOST_AUTO_TEST_CASE(pass_dynamic_arguments_to_the_base_base_with_gap)
 			}
 			uint public m_i;
 		}
-		contract Base1 is Base(3) {}
+		contract Base1 is Base {
```

Comment:
Here again, this test now tests that passing on arguments to a base that is two steps away in the linearized inheritance hierarchy, I also think this was the intention.